src/main/java/com/hcl/appscan/jenkins/plugin/scanners/Scanner.java

```diff
@@ -43,15 +43,14 @@ public String getTarget() {
 	protected String resolvePath(String path, VariableResolver<String> resolver) {
 		//First replace any variables in the path
 		path = Util.replaceMacro(path, resolver);
-		
+
 		//If the path is not absolute, make it relative to the workspace
-		File file = new File(path);
-		if(!file.isAbsolute()) {
-			String targetPath = "${WORKSPACE}" + File.separator + file.getPath();
+		if(!(path.equals("")) && ((!(path.startsWith("/"))) && (!(path.startsWith("\\"))) && (!(path.startsWith("C:\\"))) && (!(path.startsWith("D:\\"))))){
```

Comment:
Since the drive letter on Windows could be any letter, not just "C" or "D", we would be better off using a regex to determine if we've been given an absolution path or not. Something like the following:
```
Pattern pattern = Pattern.compile("^(\\\\|/|[a-zA-Z]:\\\\)");
if(!pattern.matcher(path).find()) {
    path = "${WORKSPACE}" + File.separator + path;
}
return path;
```

Reply:
Added the regular expression sequence in the scanner class.